src/Symfony/Component/OpenApi/Compiler/DocumentationCompiler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\OpenApi\Compiler;
+
+use Symfony\Component\OpenApi\Configurator\DocumentationConfigurator;
+use Symfony\Component\OpenApi\Documentation\DocumentationInterface;
+use Symfony\Component\OpenApi\Loader\ComponentsLoaderInterface;
+use Symfony\Component\OpenApi\Model\OpenApi;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ * @author Selency Team <tech@selency.fr>
+ */
+class DocumentationCompiler implements DocumentationCompilerInterface
+{
+    /**
+     * @param iterable<ComponentsLoaderInterface> $componentsLoaders
+     */
+    public function __construct(private iterable $componentsLoaders = [])
+    {
+    }
+
+    public function compile(DocumentationInterface $doc): OpenApi
+    {
+        // Instanciate root configurator
```

Comment:
Typo here. this is `Instantiate`.
but I would remove the comment entirely as the next line is clear enough